kudosaurus.go

```diff
@@ -0,0 +1,52 @@
+package kudosaurus
```

Comment:
that's how the code should be structured 👍 (all entities exactly where they should be)

Reply:
😊 

I think that's something TheMacies suggested.